byron/chain/formal-spec/blockchain-spec.tex

```diff
@@ -1124,18 +1157,30 @@ \section{Blockchain Extension}
 \end{figure}
 
 \begin{figure}[ht]
+  \emph{Chain extension environments}
+  \begin{equation*}
+    \CEEnv
+    = \left(
+      \begin{array}{r@{~\in~}lr}
+        k & \mathbb{N} & \text{chain stability parameter}\\
+        t & \left[\frac{1}{5}, \frac{1}{4}\right] & \text{block signature count threshold}\\
+        \var{ngk} & \mathbb{N} & \text{number of genesis keys}\\
+        \var{s_{now}} & \Slot & \text{current slot}
+      \end{array}\right)
+  \end{equation*}
+
+
   \emph{Chain extension states}
   \begin{equation*}
     \CEState =
     \left(
       \begin{array}{r@{~\in~}lr}
-        \var{s_{last}} & \Slot & \text{Slot of the last seen block} \\
-        \var{elens} & \Epoch \partialf \SlotCount & \text{Historical epoch lengths} \\
-        \var{sgs} & \seqof{\VKeyGen} & \text{Last signers}\\
-        \var{h} & \Hash & \text{Current block hash} \\
+        \var{s_{last}} & \Slot & \text{slot of the last seen block} \\
+        \var{sgs} & \seqof{\VKeyGen} & \text{last signers}\\
+        \var{h} & \Hash & \text{current block hash} \\
         \var{utxo} & \UTxO & \text{UTxO} \\
-        \var{us} & \UPIState & \text{Update interface state} \\
-        \var{ds} & \DIState & \text{Delegation state}
+        \var{ds} & \DIState & \text{delegation state}\\
+        \var{us} & \UPIState & \text{update interface state} \\
```

Comment:
My favourite change is switching the order of these two ;)

Reply:
Yeah, it made my eyelid twitch :P 

Not only we bring consistency, but everybody knows that delegation comes before the update :D